.github/workflows/validate_packs.yml

```diff
@@ -1,281 +0,0 @@
-name: Validate quickstarts
-
-on:
-  workflow_run:
-    workflows: ["Validation Gate"]
-    types:
-      - completed
-
-env:
-  COMMIT_SHA: ${{ github.event.workflow_run.head_commit.id }}
-
-jobs:
-  ensure-quickstart-ids-are-unique:
```

Comment:
I would argue that we would still want this job here. We don't currently have an in-house way to verify this.